src/SDKs/CognitiveServices/data-plane/Language/Microsoft.CognitiveServices.Language/Microsoft.CognitiveServices.Language.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.CognitiveServices.Language</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>Microsoft.CognitiveServices.Language</AssemblyName>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Text Analytics API;Text Analytics;REST HTTP client;netcore451511</PackageTags>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
@DavidLiCIG please remove GeneratePackageOnBuild, we have specific targets to build pacakges.
The target that we use is /t:CreateNugetPackage to create nuget packages, having it part of build system hampers the CI perf.